Warehouse.Api.Auth/Data/RefreshTokenRepository.cs

```diff
@@ -1,18 +1,18 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
 using MongoDB.Driver;
-using MongoTutorial.Core.Interfaces.Repositories;
-using MongoTutorial.Domain;
+using Warehouse.Core.Interfaces.Repositories;
+using Warehouse.Domain;
 
-namespace MongoTutorial.Data.Repositories
+namespace Warehouse.Api.Auth.Data
 {
     public class RefreshTokenRepository : IRefreshTokenRepository
     {
         private readonly IMongoCollection<RefreshToken> _tokenCollection;
 
         public RefreshTokenRepository(IMongoClient client)
         {
-            var db = client.GetDatabase("Warehouse");
+            var db = client.GetDatabase("Warehouse_refreshTokens");
```

Comment:
You don't need to add Warehouse prefix to the database name. We need the service name instead to know what service the db belongs to